Documents/start_contribution.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to JDI Light
+
+## Step 1: Clone the Repository 
+
+First, you'll need to clone the repository to your local machine. You can do this with the following command: git clone https://github.com/jdi-testing/jdi-light.git
+
+## Step 2: Navigate to the Project Directory 
```

Comment:
I think it is too obvious step, may be for previous step add that open that project in IDE (most of them have feature to create a new project from VCS)

Reply:
added